test/test_proxy_tensor.py

```diff
@@ -931,10 +931,11 @@ def forward(self, a_1):
     return div""")
 
         r = str(make_fx(f, tracing_mode="symbolic", decomposition_table=decomposition_table)(torch.empty(4)).code).strip()
+        print('DEBUG:', r)
```

Comment:
don't forget to remove when done. Also, you do know about `EXPECTTEST_ACCEPT` right

Reply:
Oh nice :)